From de1330f2fa4cc9d218e695cf0ecbcf657ab96867 Mon Sep 17 00:00:00 2001
From: Wentao Zhang <wentaoz5@illinois.edu>
Date: Sun, 25 Aug 2024 22:55:03 -0500
Subject: [PATCH v2 2/5] modfinal: disable llvm-cov instrumentation

Disable instrumentation in the same areas that were disabled for
kernel/gcov/

Signed-off-by: Wentao Zhang <wentaoz5@illinois.edu>
Reviewed-by: Chuck Wolber <chuck.wolber@boeing.com>
Tested-by: Chuck Wolber <chuck.wolber@boeing.com>
---
 scripts/Makefile.modfinal | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scripts/Makefile.modfinal b/scripts/Makefile.modfinal
index 306a6bb86..7e4a2cead 100644
--- a/scripts/Makefile.modfinal
+++ b/scripts/Makefile.modfinal
@@ -22,6 +22,7 @@ __modfinal: $(modules:%.o=%.ko)
 modname = $(notdir $(@:.mod.o=))
 part-of-module = y
 GCOV_PROFILE := n
+LLVM_COV_PROFILE := n
 KCSAN_SANITIZE := n
 
 quiet_cmd_cc_o_c = CC [M]  $@
-- 
2.45.2

